client/packages/common/src/ui/icons/Link.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const LinkIcon = (props: SvgIconProps): JSX.Element => {
```

Comment:
this looks great! can you also add to the storybook please? (`Icon.stories.tsx`)

Reply:
Thanks done